docs/src/main/paradox/server-side/websocket-support.md

```diff
@@ -143,3 +143,35 @@ testing of the WebSocket handler's behavior if the request was accepted.
 @@@
 
 See the @github[full routing example](/docs/src/test/java/docs/http/javadsl/server/WebSocketCoreExample.java).
+
+## Automatic keep-alive Ping support
+
+For long running websocket connections it may be beneficial to enable automatic heartbeat using `Ping` frames.
```

Comment:
Maybe add some reason why it would be beneficial? For newbies and search engine hits, when someone searches for "akka http web socket dropped".

Reply:
okey will add